spec/factories/user.rb

```diff
@@ -1,4 +1,8 @@
 FactoryGirl.define do
   factory :user do
+    name     "Some User"
+    email    "user@example.com"
+    password "password123"
+    password_confirmation "password123"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.